dev/ctia/dev/split_tests.clj

```diff
@@ -167,9 +167,7 @@
         example-nses (-> example-timings keys sort vec)]
     (doseq [nsplits (range 1 15)]
       (let [all-splits (for [id (range nsplits)]
-                         (binding [*out* (java.io.PrintWriter.
-                                           ;; JDK11+
-                                           (java.io.OutputStream/nullOutputStream))]
+                         (binding [*out* (java.io.PrintWriter. "/dev/null")]
```

Comment:
this does not work on windows (shall it be?).
BTW, we need to properly identify which JDK we want to support, 11 is the last LTS, 8 is not maintained anymore.

Reply:
I'd say that current consensus implcitily goes along these lines: https://github.com/threatgrid/iroh/issues/4211#issuecomment-697433583

I'll fix the /dev/null to a proper `reify` - coincidentally I have it at hand (and saved it as a handy emacs `yasnippet` ;p)